metrics/network/remote_network/remote-networking-test-common.sh

```diff
@@ -170,3 +170,28 @@ function clean_environment {
 	$DOCKER_EXE swarm leave --force
 	ssh "$ssh_user"@"$ssh_address" "$DOCKER_EXE swarm leave --force"
 }
+
+# This function starts an iperf3 server container
+function direct_start_server {
```

Comment:
As this function has a hard wired iperf3 in it, maybe we should name the function to reflect that - or, add a generic start function that can then take an argument of what to run?